src/items/items.hpp

```diff
@@ -435,6 +442,35 @@ class Items {
 		return dummys;
 	}
 
+	static const std::string getAugmentNameByType(AugmentTypes_t augmentType) {
+		phmap::flat_hash_map<AugmentTypes_t, std::string> map = {
+			{ AUGMENT_COOLDOWN, "cooldown" },
+			{ AUGMENT_CRITICALHITDAMAGE, "critical extra damage" },
+			{ AUGMENT_INCREASED_DAMAGE, "Increased Damage" },
+			{ AUGMENT_LIFELEECHAMOUNT, "life leech" },
+			{ AUGMENT_MANALEECHAMOUNT, "mana leech" },
+			{ AUGMENT_POWERFUL_IMPACT, "Powerful Impact" },
+			{ AUGMENT_STRONG_IMPACT, "Strong Impact" },
+		};
+
+		const auto it = map.find(augmentType);
+		if (it != map.end()) {
+			return it->second;
+		}
+
+		return "unknown type";
+	}
+
+	static const bool isAugmentWithoutValueDescription(AugmentTypes_t augmentType) {
+		std::vector<AugmentTypes_t> vector = {
```

Comment:
As previously mentioned, avoid creating a new vector during each function call to optimize performance.
```suggestion
		static std::vector<AugmentTypes_t> vector = {
```